apps/opik-backend/src/main/java/com/comet/opik/domain/FeedbackScoreDAO.java

```diff
@@ -153,6 +155,47 @@ AND entity_id IN (
             ;
             """;
 
+    private static final String SELECT_FEEDBACK_SCORE_NAMES = """
+            SELECT
+                distinct name
+            FROM (
+                SELECT
```

Comment:
Seems like the nested query here is unnecessary if the inner query is added with `distinct`. Unless I'm missing something.

Reply:
The issue here is that rows can be duplicated in Clickhouse. When there are updates, Clickhouse takes some time to remove the previous "version of the row". If you check, we always do this order by and limit by id

Reply:
Makes sense